tests/drum/test_units.py

```diff
@@ -367,6 +443,344 @@ def test_yaml_metadata_missing_fields(tmp_path, config_yaml, request, test_case_
         read_model_metadata_yaml(tmp_path)
 
 
+@pytest.mark.parametrize(
+    "hyper_param_metadata",
+    [
+        "complete_int_hyper_param",
+        "complete_float_hyper_param",
+        "complete_string_hyper_param",
+        "complete_select_hyper_param",
+        "complete_multi_hyper_param",
+    ],
+)
+@pytest.mark.parametrize(
+    "param_name_value, error",
+    [
+        ("_param", "The parameter name should not start or end with the underscore.",),
+        ("param_", "The parameter name should not start or end with the underscore.",),
+        ("1" * (PARAM_NAME_MAX_LENGTH + 1), "String is longer than 64 characters",),
```

Comment:
Maybe we should make this message 'Parameter name is longer than 64 characters' so they don't get confused with a string value

Reply:
A trafaret validator based on t.String is created to wrap the error message
https://github.com/datarobot/datarobot-user-models/blob/e7ba94491c54a5091532ae3a8529eb46f06b4cb5/custom_model_runner/datarobot_drum/drum/model_metadata.py#L27-L41

https://github.com/datarobot/datarobot-user-models/blob/e7ba94491c54a5091532ae3a8529eb46f06b4cb5/tests/drum/test_units.py#L492-L500